runtime/src/main/java/io/quarkiverse/logging/splunk/SplunkConfig.java

```diff
@@ -35,7 +35,8 @@ public class SplunkConfig {
     /**
      * Splunk HEC endpoint base url.
      * <p>
-     * The actual endpoint is expected at path /services/collector/events/1.0
+     * The actual endpoint is expected at path /services/collector/events/1.0 for JSON events or
+     * /services/collector/event/1.0 for raw events.
```

Comment:
Isn't it services/collector/raw/1.0 ?

Reply:
Ops, I copied the wrong link. Docs state that `/services/collector/raw` and `/services/collector/raw/1.0` are equivalent; I used `/services/collector/raw` because that's what `com.splunk.logging.HttpEventCollectorSender` is using